SpinCore_pp/save_data.py

```diff
@@ -0,0 +1,70 @@
+import h5py
+import os
+from pyspecdata import *
+import pyspecProcScripts
+import subprocess
+
+
+def save_data(dataset, my_exp_type, config_dict, counter_type):
+    """save data to an h5 file with appropriately labeled nodename and performs
+    rough processing
+
+    Parameters
+    ==========
+    dataset: nddata
+        acquired data in nddata format
+    my_exp_type: str
+        directory on the share drive you want to save to
+    config_dict: dict
+        config_dict pulled from the active.ini file
+    counter_type: str
+        type of counter you are incrementing
+
+    Returns
+    =======
+    config_dict: dict
+        the updated config dict after appropriately incrementing the counter
+    """
+    target_directory = getDATADIR(exp_type=my_exp_type)
```

Comment:
actually, to keep it simple, just pass the target directory

Reply:
we decided against this in other comments